### PR TITLE
Conditional export for theme/logos

### PR DESCRIPTION
**Pull request by Fuzznimp**
## Description

This PR aims at adding more flexibility on the theme/logo export. This is most likely a temporary way of doing it.

## Type of change

- [x] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## How Has This Been Tested?

Locally and with the test suit.

## Checklist:

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

File: src/components/display/fewlines/Header/Header.tsx [hunk 1]
```diff
@@ -1,12 +1,16 @@
 import React from "react";
 import styled from "styled-components";
 
-import fewlinesLogo from "@src/assets/logo-fewlines-2020.png";
+import decatLogo from "@src/assets/logo/decathlon-logo.png";
+import fewlinesLogo from "@src/assets/logo/logo-fewlines-2020.png";
+import { config } from "@src/config";
 
 export const Header: React.FC = () => {
+  const logo = config.connectTheme === "decathlon" ? decatLogo : fewlinesLogo;
```
Comment (Fenntasy) [inline src/components/display/fewlines/Header/Header.tsx:9]: couldn't we have the logo be in the theme directly?

File: src/config.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ import OAuth2Client, { OAuth2ClientConstructor } from "@fwl/oauth2";
 
 type Config = {
   connectDomain: string;
+  connectTheme: string;
```
Comment (Fenntasy) [inline src/config.ts:5]: shouldn't it be optional?

File: src/components/display/fewlines/Header/Header.tsx (hunk 1 above)
Comment (Fuzznimp) [reply src/components/display/fewlines/Header/Header.tsx:9]: Good idea.

File: src/config.ts (hunk 2 above)
Comment (Fuzznimp) [reply src/config.ts:5]: I think leaving this as it is, but changing the field's value attribution like this should be better:
```ts
config.connectTheme = process.env.CONNECT_ACCOUNT_THEME || "fewlines";
```
What do you think?

Comment (Fenntasy) [reply src/config.ts:5]: 👍

File: src/config.ts [hunk 3]
```diff
@@ -40,6 +42,7 @@ const config: Config = {
 
 function handleEnvVars(): void {
   config.connectDomain = process.env.CONNECT_ACCOUNT_DOMAIN || "";
+  config.connectTheme = process.env.CONNECT_ACCOUNT_THEME || "";
```
Comment (Fenntasy) [inline src/config.ts:45]:
```suggestion
  config.connectTheme = process.env.CONNECT_ACCOUNT_THEME || "fewlines";
```

File: src/components/display/fewlines/Icons/EditIcon/EditIcon.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
```
Comment (Fenntasy) [inline src/components/display/fewlines/Icons/EditIcon/EditIcon.stories.tsx:1]: weird that this file poped up

**Review by Fenntasy — APPROVED** [review]

**Pull request merged**